packages/wp-source/src/libraries/handlers/author.ts

```diff
@@ -13,6 +13,15 @@ const authorHandler: Handler = async ({
   const { path, page, query } = parse(route);
   const { slug } = params;
 
+  // Add the attributes that should be present even if fetch fails or we throw a ServerError below
```

Comment:
Good one 👍 

Yes, after this PR I expect people to use
```js
const { path } = state.source.get(someLink);
```
instead of
```js
const { path } = libraries.source.parse(someLink);
```